NOTICE.txt

```diff
@@ -9,3 +9,10 @@ This product includes software from the QT project (BSD, 3-clause).
 
 This product includes software from HowardHinnant's date library (MIT License).
 * https://github.com/HowardHinnant/date/tree/master
+
+This product includes software from the The Arrow project (Apache 2.0).
```

Comment:
Thanks for adding this! A requirement of the ASL 2.0 is that the NOTICE file of the project is reproduced e.g. https://github.com/apache/arrow/blob/main/NOTICE.txt#L60-L71

Arrow lists the specific files used/based on from other projects in detail in LICENSE.txt e.g. the same kudu bits i highlighted in the notice file have this corresponding entry: https://github.com/apache/arrow/blob/main/LICENSE.txt#L754-L760

I think these additional entries are mostly to provide the other (non ASL 2.0) licenses as required, so might not really be necessary here.

Reply:
@assignUser Thanks for your detailed explanations. I removed the Apache 2.0. Can you help to review again? Thanks.